### ISSUE TITLE
showing codes together with other concept info

### ISSUE THREAD

**Issue by catecara** [NatLibFi/Skosmos#627]
This is not a bug report but a request for features improvement. 

Currently, codes are shown together with the pref label, and not shown in description of the concept. While I find it useful to have it in the pref label, I think the code should also be shown as a separate piece of info of the concept. In case, it could be something to set as a preference. For my work on stat classification, I would definitely have it in both places.

**Comment by osma** [comment]
This would be useful in particular when a concept has multiple notations, for example chemical codes according to different coding schemes, differentiated using RDF data types:

```
      skos:notation "C=CCBr"^^c:SMILES ;
      skos:notation "InChI=1S/C3H5Br/c1-2-3-4/h2H,1,3H2"^^c:InChI .
```

**Comment by osma** [comment]
This was implemented in PR #1198 which went into the Skosmos 2.12 release. Closing this issue as well.
